src/hooks/transactions/useSignMultipleTransactions.tsx

```diff
@@ -109,9 +110,15 @@ export function useSignMultipleTransactions({
     const { tokenId } = transactionTokenInfo;
     const receiver = transaction.getReceiver().toString();
     const sender = transaction.getSender().toString();
-    const isLoggedInWithDifferentAccount = sender && address !== sender;
-
-    // Prevent signing transactions with different account
+    const senderAccount = await getAccount(sender);
+    const isLoggedInWithDifferentAccount =
```

Comment:
could be extracted and used also on `useSignTransactions.tsx` file at line: 264.